elasticsearch/Dockerfile

```diff
@@ -1,70 +1,80 @@
-FROM rhel7.3:7.3-released
+FROM rhel7:7-released
 
 MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
 
 EXPOSE 9200
 EXPOSE 9300
 USER 0
 
-ENV ES_CLOUD_K8S_VER=2.4.4.01 \
-    ES_CONF=/usr/share/java/elasticsearch/config \
+ENV ES_CLOUD_K8S_VER=5.5.2 \
+    ES_CONF=/etc/elasticsearch/ \
     ES_JAVA_OPTS="-Dmapper.allow_dots_in_name=true" \
-    ES_HOME=/usr/share/java/elasticsearch \
+    ES_HOME=/usr/share/elasticsearch \
     ES_VER=5.5.2 \
     HOME=/opt/app-root/src \
     INSTANCE_RAM=512G \
     JAVA_VER=1.8.0 \
     NODE_QUORUM=1 \
-    OSE_ES_VER=2.4.4.20 \
-    PROMETHEUS_EXPORTER_VER=2.4.4.1 \
+    OSE_ES_VER=5.5.2.1 \
+    PROMETHEUS_EXPORTER_VER=5.5.2.0 \
     PLUGIN_LOGLEVEL=INFO \
     RECOVER_AFTER_NODES=1 \
     RECOVER_EXPECTED_NODES=1 \
     RECOVER_AFTER_TIME=5m \
     RELEASE_STREAM=prod
 
-ARG ES_CLOUD_K8S_VER=2.4.4.01
-ARG OSE_ES_VER=2.4.4.16
+ARG ES_CLOUD_K8S_VER=5.5.2
+ARG OSE_ES_VER=5.5.2.1
 ARG ES_CLOUD_K8S_URL
 ARG OSE_ES_URL
 ARG PROMETHEUS_EXPORTER_URL
+ARG LOCAL_REPO
 
-LABEL io.k8s.description="Elasticsearch container for EFK aggregated logging storage" \
-      io.k8s.display-name="Elasticsearch ${ES_VER}" \
-      io.openshift.expose-services="9200:https, 9300:https" \
-      io.openshift.tags="logging,elk,elasticsearch" \
-      com.redhat.component=logging-elasticsearch-docker \
-      architecture=x86_64 \
-      name="openshift3/logging-elasticsearch" \
-      version="v3.7.0" \
-      release="0.127.0.0"
+ADD sources /tmp/lib/
+WORKDIR /tmp/lib
+RUN while read -r hash file ; do \
+      curl -s -o $file http://pkgs.devel.redhat.com/repo/rpms/logging-elasticsearch-docker/$file/$hash/$file ; \
+    done <sources
 
-RUN yum-config-manager --enable rhel-7-server-ose-3.7-rpms \
+RUN if [ -n "${LOCAL_REPO}" ] ; then \
+     curl -s -o /etc/yum.repos.d/local.repo ${LOCAL_REPO} ; \
+    fi
+
+RUN yum-config-manager --enable rhel-7-server-ose-3.8-rpms \
```

Comment:
3.9 or maybe 3.10?

Reply:
from my conversations with others familiar with brew, this line is either modified directly or ignored completely so going to leave as is